### ISSUE TITLE
Report when there are executables that are not put on the PATH

### ISSUE THREAD

**Issue by ferventcoder** [chocolatey/choco#691]
If a package installs files into program files or somewhere else that are not shimmed, but the PATH is not updated (or already has those files on the path), report that discrepancy to the user.

Depends on #690.

**Comment by ferventcoder** [comment]
Requested by @gpakosz

**Comment by pauby** [comment]
Thanks for raising this issue. After reviewing it, it's not something we expect to prioritise in the foreseeable future. We’re closing it to keep our backlog focused, but we’re open to revisiting it in the future if priorities shift or new interest arises. You are welcome to comment here or open a new issue if you'd like to discuss it further down the line.
